src/Sim.tsx

```diff
@@ -258,13 +283,37 @@ export class Space {
       // }
     });
   }
+  
 
   public startRenderLoop(): void {
     this.engine.runRenderLoop(() => {
+      this.scene.executeOnceBeforeRender(() => this.scene.getPhysicsEngine().setGravity(new Babylon.Vector3(0,-9.8 * 10,0)),500);
```

Comment:
Dunno if it's expensive but I think this is getting called once per frame. Are the gravity changes between `-9.8`, `0`, and `-9.8 * 10` still needed?

Reply:
One way to optimize this would be to store the lambda as a instance variable in the class (like a regular React callback) and make the gravity a constant/global. With those two allocations not happening every frame it should basically be free.

Reply:
This should be fixed now in the new commit. Seems to work better.